ui/src/models/versions.js

```diff
@@ -0,0 +1,48 @@
+export default class Version {
```

Comment:
If we add in a version model, we should also stay consistent and add a model for apps as well.

Reply:
Is there a particular usecase/reason to use a model?

Reply:
I find it a bit cleaner than referencing array[x], particularly as we are now going to be setting the values for the disabled boolean and the disabled message. That's really the only value we get from it so we don't _have_ to use it.